### PR TITLE
Mining points are back

### PR DESCRIPTION
**Pull request by MCterra10**
## About The Pull Request
This PR reinstates mining points as the currency earned from submitting ore to the ORM. Shaft Miners paygrade has also been re-re-balanced to account for this. In addition I have readded point transfer cards and space cash to the mining equipment vendor.
Since vendors no longer use credits, I have readded mining conscription kits to cargo as a way to recruit new miners without needing an actual miner get the kit.

## Why It's Good For The Game
The original programmer who was leading the charge on balancing lavaland with the station and the main proponent for integrating mining points into the station economy has since almost entirely left /tg/, leaving mining points/credits in a highly unbalanced state. 

As a result we have had several PRs nerfing or removing survival medipens due to heads using their budgets to buy large quantities. I intend to resolve the cause of these issues in this PR rather than fix the symptoms, and I feel that bringing back mining points is the best way the achieve this until such times as someone steps up to rework lavaland.

Oranges "Approved"
oranges

## Changelog
:cl:
add: Shaft Miner Conscription Kit cargo crate available again
add: Point transfer cards and space cash can now be bought from the vendor again
add: Mining points readded
balance: Miners have higher income
/:cl:

### CODE REVIEW

File: code/modules/mining/machine_redemption.dm [hunk 1]
```diff
@@ -37,21 +38,22 @@
 
 /obj/machinery/mineral/ore_redemption/RefreshParts()
 	var/ore_pickup_rate_temp = 15
+	var/point_upgrade_temp = 1
 	var/sheet_per_ore_temp = 1
 	for(var/obj/item/stock_parts/matter_bin/B in component_parts)
-		sheet_per_ore_temp = 0.65 + (0.15 * B.rating)
-	for(var/obj/item/stock_parts/micro_laser/L in component_parts)
```
Comment (Qustinnus) [inline code/modules/mining/machine_redemption.dm]: What is this for

Comment (Qustinnus) [reply code/modules/mining/machine_redemption.dm]: As in, why are you removing it

**Review by ghost — CHANGES_REQUESTED** [review]
Looks mostly fine, maybe allow mining points to be converted to money (and not the other way around) so you can still earn money from getting points.

File: code/modules/mining/machine_redemption.dm (hunk 1 above)
Comment (Qustinnus) [reply code/modules/mining/machine_redemption.dm]: Mmh nevermind, you're using them for amount of points gained, I think matter bins shouldn't give you the amount laser + bins would give though.

Comment (MCterra10) [reply code/modules/mining/machine_redemption.dm]: Yeah that was swapped out for another bonus. Specifically the same bonus as back when mining points originally existed.
Also mining points can be converted to credits, as I have readded the space credits item to the vendor. I'll actually update the PR to mention that

**Comment by kriskog** [comment]
Remove budgets instead, they fuck up more balance-wise than just the mining related stuff. Seriously, just get rid of them. They were bad when they were added, and after a few nerfs they are still bad and any cool and exciting uses for economy suffer as a result. This is a prime example of one such use, letting you get some expensive gamer gear with your hard earned cash.

If you do choose to keep going down this path though, readd the conscription kit crate to cargo so it is once again possible to buy one for money. Removed in #45089.

**Comment by ATH1909** [comment]
https://www.youtube.com/watch?v=_J6-3l3hCm0

Also, +1 to (just) the "readd the conscription kit crate to Cargo" part of kriskog's comment.

**Comment by MCterra10** [comment]
Should I do that in this PR or make another

**Comment by MCterra10** [comment]
You know what, fuck it. I'll add it back here.

**Comment by Arathianen** [comment]
+1 for removing the free money budgets. It's the main issues with balance. Heads getting 5 grand every x minutes is a terrible idea.

Or maybe keep them with some initial cash (around a grand) and only get money from activities like sci bombs, gulaging and so on.

**Comment by Akrilla** [comment]
Gross.

**Comment by GuillaumePrata** [comment]
The change from Mining Points to cash made stuff more practical, I think the budget cards are the real problem with economy now, not the mining items themselves.

**Comment by Ratvar** [comment]
All of those are a problem: mining point removal, letting everyone access dumb superheals, miners needing superheals to do their super job unrelated to ss13 and departmental budgets existing at all. Good thing this PR fixes one of those.

**Comment by shellspeed1** [comment]
Might be worth adding a ORM to the mining base for people who are mining as nonminers since it's public access now and all that.

**Review by 81Denton — CHANGES_REQUESTED** [review]
Edit: Disregard this 20 IQ review

**Comment by Learner10** [comment]
This change makes issues with player fun. You need to make a way for anyone who brings in ore to acquire said mining points, otherwise the various public access people who go to Lavaland to get a hardsuit, or self-conscript themselves won't even have a reason to go. 

I'd feel less bothered with simply hijacking the ORM and yeeting it down to Lavaland just to get the points, and I'd probably not even get yelled at by the crew for it either. Why? Because the ORM still only accepts ore from one side, and any ore directly deposited still doesn't properly relinquish the points/money. If I happened to destroy the coin minter, I wouldn't feel bothered by it, since it'd be right back to being a pointless antiquated piece of machinery.

So... sure, you could probably change this by just making the ORM accept from all sides.

**Comment by MCterra10** [comment]
@81Denton I tested this never noticed an issue with claiming points... Are you aware that you no longer insert your ID into the ORM to claim points? It claims fine when in the ID slot

**Comment by MCterra10** [comment]
That change was made in #45693

**Comment by 81Denton** [comment]
O yeah. My bad. Uhhh disregard my review and I'll merge this as usual then.

**Review by 81Denton — APPROVED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
no one will ever fix it proper if you bandaid it like this.

**Comment by Qustinnus** [comment]
@ExcessiveUseOfCobblestone noone is fixing it now either.

**Comment by ExcessiveUseOfCobblestone** [comment]
none of the items are gamebreaking and if there's such a necessity for this PR then people should pursue a proper fix either on econ or the questionable item.

This is coming from someone who is disadvantaged by having it based on econ since medipens in particular are pretty anti-mymedrework.

Does it suck? Maybe. We should work to make it not suck if it does.

**Comment by Fikou** [comment]
it was cool being able to mine stuff and get points for it to buy stuff with public mining, maybe some variant of the orm that only can deposit mats and collect points would be cool

**Comment by OverlyGeneric** [comment]
it was stupid to remove mining points, to begin with.

mining isn't balanced for the station and that is a whole another problem on its own.

I don't see that mining should be balanced for the station but at least make certain aspects of it more difficult to do like what Whoneedsspace was doing.

that being said, we should have never even allowed people to buy things with their own money, round start, from the mining vending machine.

it was always meant to be a reward for miners who did their job instead of rushing megafauna but the removal dissuaded miners from that.

**Comment by CRITAWAKETS** [comment]
Dont forget to bring back the micro-laser's old function back to the ORM where it increased mining point gain.

**Comment by Whoneedspacee** [comment]
cobby, mining points were fine before.

at least this gives incentive to use the ORM, before I’d just use the coin press on diamonds and get even more since you’d get like 5x the amount of money.

mining items should stay with the incentive to mine, not make money.

on that note if we got some more mining items that dealt with the unique natures of lavaland that’d be good as well.

i’ve wanted to add descriptions of the items in the ORM for a while anyways so people would stop ignoring items like the jaunter, maybe it’s about time for that.

**Comment by Akrilla** [comment]
Annnnnd we're back to mining being their own little island.

**Comment by 81Denton** [comment]
Mining content really needs to be rebalanced first before it's unleashed upon the rest of the station.

**Comment by ExcessiveUseOfCobblestone** [comment]
> cobby, mining points were fine before.

Mining (and consequently mining points) is not "fine", the idea that everything has to scale because someone decided to not balance it preemptively, resulting in a vendor that has powerful loot gated behind an arbitrary side-economy because it's literally TOO powerful for common use is stupid. 

Mining items should stay with the incentive to mine yes, but shouldn't be so powerful that every job wants to use the mining content to begin with.

Medical items should stay with the incentive to help patients, but you don't see me putting in rapid healing hyposprays and locking the vendor behind how much damage a user has healed despite medbay being the most targeted department in nearly all modes (and players don't have the luxury of having their moves limited to a set amount of attacks you can train against offline). 

Maybe I wouldn't be so irked by this vendor if stuff like the medical pen was balanced to only work in lavaland to begin with. The whole sandbox thing is super annoying because it only works one way.

**Comment by Whoneedspacee** [comment]
it’s not that the loot is too powerful, it’s that it can’t be balanced to scale properly because of department budgets.

it needs to work on a seperate budget because money generally can’t buy anything fucking good besides like guns from cargo. and because you don’t want miners getting literally as much and more money than heads.

it will never be balanced to work with economy because it literally cannot fit.

**Comment by Whoneedspacee** [comment]
on another point, why shouldn’t mining content be strong? it’s not like you’re allowed to buy stimulum or buy other high end parts of jobs, the mining items are relatively expensive for the amount of ore you need to mine (except survival medipen tbh).

i just don’t agree with your seemingly philosophy that everyone should be able to access all job content instantly if they have the knowledge, it’s not the case with mining or most other jobs, and it really shouldn’t be. 

yes maybe you steal a miners id, ore, or backpack, or kill the chief engi as he’s making stimulum, but straight up getting it, nah.

**Comment by ExcessiveUseOfCobblestone** [comment]
To be clear, this is expected to return once econ gets tuned out a bit further.

> it’s not that the loot is too powerful, it’s that it can’t be balanced to scale properly because of department budgets.

if the loot wasn't good why would people be buying it so much that this becomes an issue in the first place? I might could see it being more of an issue that these vendors have seemingly infinite inventory though.

I do agree budget cards are trash but no one cba to fix them and I don't want to spin econ on top of medical/parts of combat.

> i just don’t agree with your seemingly philosophy that everyone should be able to access all job content instantly if they have the knowledge, it’s not the case with mining or most other jobs, and it really shouldn’t be.

What does this even mean? 

My philosophy is that the mining vendor should be a medium by which a user can obtain a partial amount of mining content without necessarily doing the job in the same way mining/xenobio/etc. can step on the toes of medical with self-produced healing items; Xenobio can step on the toes of mining w metals; and botany can step on the toes of chem with making certain reagents.

You're being really silly if you're trying to imply that the vendor is the source of "all (miner) content" or even a bulk of it. Maybe if you took the job as strictly mining but lol...

**Comment by Whoneedspacee** [comment]
they're literally buying one item dude, stop exaggerating.

**Comment by CRITAWAKETS** [comment]
survival medipens are actually the only thing that departments other than cargo even bought besides the luxury bar for meme builds

PKAs and hardsuits were pretty much exclusively bought by QMs because honestly what is the point if you dont mine. Just use a gun instead of a PKA (cargo is one room next to you bud) and use an EVA suit or a hardsuit from your own department.

The medipens are hot shit but guess what, this PR fixes that issue by making it so only people who mine can get the gear for you know, mining.

**Comment by ExcessiveUseOfCobblestone** [comment]
🤔 is a survival pen not part of the mining vendor loot?

> my point is, is that ores can't scale properly with economy, so you get issues like we have where someone gets a shitton of money and buys a billion survival medipens or whatever.

They can, you just refuse to manage it (which is fine, I don't blame you). Compared to the other sources of income it's probably the most controlled seeing as you're limited to the contents of lavaland.

Also this PR still allows Ores to be doubled as money makers on top of the items produced since you can get cash with your monopoly money, so no wonder it's hard to scale it properly in either direction.

> The medipens are hot shit but guess what, this PR fixes that issue by making it so only people who mine can get the gear for you know, mining.

"""mining"""

My issue with medipens is exactly that, it's meant for mining but has fairly good applications outside of mining. Rather than balance the medipen so that it's use is more geared towards actual mining (on lavaland), we restrict the vending machine entirely to miner players using yet another subeconomy that has to be maintained?

I mean I guess i'll have to fix this myself later but i haven't mined in a hot minute so hopefully someone will get to it before i do.

**Comment by kriskog** [comment]
Medipens are a problem and causes us to go with unfun shit like this? Problem solved:
```/obj/item/reagent_containers/hypospray/medipen/survival/inject(mob/living/M, mob/user)
	if(z != is_mining_level())
		to_chat(user, "The [src] makes an ominous buzzing sound and refuses to open.")
		return
	. = ..()```

**Pull request merged**